packages/template-retail-react-app/app/translations/en-US.json

```diff
@@ -511,7 +511,7 @@
     "defaultMessage": "Terms & Conditions"
   },
   "footer.message.copyright": {
-    "defaultMessage": "2021 Salesforce or its affiliates. All rights reserved. This is a demo store only. Orders made WILL NOT be processed."
+    "defaultMessage": "Salesforce or its affiliates. All rights reserved. This is a demo store only. Orders made WILL NOT be processed."
```

Comment:
@bfeister Please update the other locales too, just like how you did here with `en-US`.